scheduler/src/cook/kubernetes/api.clj

```diff
@@ -420,20 +420,26 @@
                                   (.getKey %))
                                taints-on-node)
           node-name (some-> node .getMetadata .getName)
+          ^V1NodeSpec node-unschedulable (some-> node .getSpec .getUnschedulable)
```

Comment:
`^V1NodeSpec node-unschedulable`
this doesn't look right
that's a boolean, not a node spec

Reply:
Good catch. Thanks.